wadsrc/static/zscript/ui/statusbar/doom_sbar.zs

```diff
@@ -89,18 +89,19 @@ class DoomStatusBar : BaseStatusBar
 		String image;
 		for(int i = 0; i < 6; i++) locks[i] = CPlayer.mo.CheckKeys(i + 1, false, true);
 		// key 1
-		if (locks[1] && locks[4]) image = "STKEYS6";
+		if (locks[1] && locks[4]) image = "ZTKEYS1";
 		else if (locks[1]) image = "STKEYS0";
 		else if (locks[4]) image = "STKEYS3";
+		else image = "";
```

Comment:
This line is useless, `image` is empty at this point.

Reply:
If images are added above this one for some reason, it would help with possible issues using this for that case.